paasta_tools/kubernetes_tools.py

```diff
@@ -1570,6 +1570,12 @@ def get_pod_template_spec(
         if prometheus_shard:
             labels["paasta.yelp.com/prometheus_shard"] = prometheus_shard
 
+        if system_paasta_config.get_kubernetes_add_registration_labels():
+            # Allow Kubernetes Services to easily find
+            # pods belonging to a certain smartstack namespace
+            for registration in self.get_registrations():
+                labels[f"paasta.yelp.com/registrations/{registration}"] = "true"  # type: ignore
```

Comment:
Why the type error is ignored?

Reply:
https://github.com/python/mypy/issues/7178